thermo_scenes/docs/Collect_new_dataset.md

```diff
@@ -18,11 +18,11 @@ And align the RGB boundaries with the thermal image using the slider:
 
 You must use FLIR cloud service to transfer the images to your computer:
 
-- First, select all the images and download them, this will save the MSX images in a zip file.
-- Then, download one by one the RGB DC images by opening them in the gallery.
-  This step is necessary to align the RGB and thermal image; using the MSX image to extract both will result in non-aligned RGB/thermal images.
+- First, select all the thermal images and download them, this will save the MSX images in a zip file. ![alt text](images/flir_download_thermal.png)
```

Comment:
You are selecting the general file not just thermal images, so let's remove thermal. Also can you add an alt text to the image?